### ISSUE TITLE
No AR Foundation samples

### ISSUE THREAD

**Issue by pixlhero** [juicycleff/flutter-unity-view-widget#721]
In the readme it says

> There are now two unity app examples in the unity folder, one with the default scene and another based on Unity AR foundation samples.

 However I can only find one unity project at `example/unity/DemoApp`, and even though it contains the ARFoundation packages, there are no scenes using AR at all. Are these examples moved somewhere else? If yes the readme should be updated.

**Comment by timbotimbo** [comment]
I don't think the arfoundation example was ever uploaded. That line in the readme has been there for over a year, without a second unity project being added. Probably some good intentions that never ended up happening.

The project is still current, and breaking issues with Unity or OS updates get fixed. But it is not the main focus of the creator/maintainer.

I can confirm that ARFoundation works in both Unity 2019 and 2021, but I don't have any ready made examples.

**Comment by andyshephard** [comment]
The readme does link to another GitHub repository which has an AR example. This is it:
https://github.com/juicycleff/flutter-unity-arkit-demo

**Comment by pixlhero** [comment]
@andyshephard yeah I saw that repository as well, but didn't work for me. And it seems kind of dead (no commits for 3 years, no answers to issues).
